README.rst

```diff
@@ -49,6 +49,24 @@ Much effort is underway to continue expanding and developing packages in the
 for each package, you can post issues and request new features. You can also email
 questions to `PyAnsys Support <mailto:pyansys.support@ansys.com>`_.
 
+By default, the PyAnsys package installs the core modules. The following PyAnsys packages are considered as core:
+
+- `PyAEDT`_
+- `PyDPF-Core`_
+- `PyDPF-Post`_
+- `PyMAPDL`_
+- `PyFluent`_
```

Comment:
Can we alphabetize so that a reader can quickly see if the package wanted is included? (I'd also like to make this change in the README for the PyAnsys account.)

Reply:
Will do! :)